pkg/wallet/profile.go

```diff
@@ -94,9 +109,64 @@ func (pr *profile) setKMSOptions(passphrase string, secretLockSvc secretlock.Ser
 	return nil
 }
 
-func (pr *profile) setEDVOptions(edvServerURL, vaultID string) {
-	pr.EDVServerURL = edvServerURL
-	pr.VaultID = vaultID
+func (pr *profile) setEDVOptions(opts *edvConf) error {
+	// non EDV users.
```

Comment:
Question: can each user have a different type of storage?

Reply:
Yes